tests/ltp/ltp-batch1/ltp_disabled_tests.txt

```diff
@@ -261,10 +261,10 @@
 #/ltp/testcases/kernel/syscalls/fstatat/fstatat01
 #/ltp/testcases/kernel/syscalls/fstatfs/fstatfs01
 #/ltp/testcases/kernel/syscalls/fstatfs/fstatfs02
-#/ltp/testcases/kernel/syscalls/fsync/fsync01
-#/ltp/testcases/kernel/syscalls/fsync/fsync02
-#/ltp/testcases/kernel/syscalls/fsync/fsync03
-#/ltp/testcases/kernel/syscalls/fsync/fsync04
+/ltp/testcases/kernel/syscalls/fsync/fsync01
```

Comment:
Why are we disabling fsync tests? We are loosing total syscall test coverage here. As we discussed we cannot use loop device file system and we use root file system to enable these tests. What solution do you recommend to run these  tests?

Reply:
I've provided an explanation, and moved the changes to the CI pipeline to a separate PR.